x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/privilege_monitoring_data_client.ts

```diff
@@ -236,7 +236,16 @@ export class PrivilegeMonitoringDataClient {
   }
 
   async getEngineStatus() {
-    const engineDescriptor = await this.engineClient.get();
+    const findResponse = await this.engineClient.find();
+    const engineDescriptor =
+      findResponse.total > 0 ? findResponse.saved_objects[0].attributes : undefined;
+
+    if (!engineDescriptor) {
+      return {
+        status: PRIVILEGE_MONITORING_ENGINE_STATUS.NOT_INSTALLED,
+        error: undefined,
```

Comment:
This isn't strictly required for the bug fix, but it is a good way to prevent similar bugs in the future.